### ISSUE TITLE
egl_render rendering planned ?

### ISSUE THREAD

**Issue by rafaelSorel** [OtherCrashOverride/c2play#28]
Hello, 

Is it possible to add flagged video egl_rendering using amvideocapture or ion ?

Thanks,

**Comment by OtherCrashOverride** [comment]
The Mali-450 GPU does not support ARM Framebuffer Compression (AFBC).  This means its not possible to process 4K@60 video using the GPU due to memory bandwidth limitations.  The video output layer currently used does support AFBC and works without issue at 4K@60.  Due to this, I have no plans to add EGL rendering.

**Comment by rafaelSorel** [comment]
Yes I perfectly get this part , I want just to experiment the 1080p@60fps using EGL, thats why I have asked for this.
If you confirm that I can do it using amlcapture, with a copy to EGL image , it is ok for me ?

**Comment by OtherCrashOverride** [comment]
An example is here:
https://github.com/OtherCrashOverride/CXVideoCube/commits/ce06e11f770f2e8aaabae1c81336e9cd0fdf368f

Relevant info here:
https://forum.odroid.com/viewtopic.php?f=136&t=21948&start=200#p203796

TL;DR = It can be done because it has been done, but the example is not up to date.  Its meant for developer guidance only.
